Add AdmissionPolicy documentation (#90)
kubewarden/docs: src/quick-start.md

> Kubewarden has two main components which you will interact with:

Reviewer: ```suggestion
Kubewarden has three main components which you will interact with:
```

Author: done in https://github.com/kubewarden/docs/pull/90/commits/3884c372cb0809d18ac9e102d03aef8387e41992

| | | - `Ignore`: an error calling the webhook is ignored and the API request is allowed to continue |
| | | - `Fail`: an error calling the webhook causes the admission to fail and the API request to be rejected |

> **NOTE:** The  `ClusterAdmissionPolicy` resources are registered with a `*` webhook `scope`, which means that registered webhooks will forward all requests matching the given `resources` and `operations` -- either namespaced (in any namespace), or cluster-wide resources.

### AdmissionPolicy

> `AdmissionPolicy` is a namespace-wide resource. This means that its resources are registered with a `Namespaced` webhook `scope` that will forward all requests matching the given `resources` and `operations` within the `AdmissionPolicy` namespace. Other than that, there are no functional differences between the `AdmissionPolicy` and `ClusterAdmissionPolicy` resources.

Reviewer 2: I think this is exposing low level details about how kubernetes's webhook registration works. Something that end users do not care/might now even know about.

What about something like that:

```suggestion
`AdmissionPolicy` is a namespace-wide resource. The policy will process only the requests that are targeting the Namespace where the `AdmissionPolicy` is defined. Other than that, there are no functional differences between the `AdmissionPolicy` and `ClusterAdmissionPolicy` resources.
```

| | | - `Ignore`: an error calling the webhook is ignored and the API request is allowed to continue |
| | | - `Fail`: an error calling the webhook causes the admission to fail and the API request to be rejected |

> **NOTE:** The  `ClusterAdmissionPolicy` resources are registered with a `*` webhook `scope`, which means that registered webhooks will forward all requests matching the given `resources` and `operations` -- either namespaced (in any namespace), or cluster-wide resources.

### AdmissionPolicy

> `AdmissionPolicy` is a namespaced resource. Its resources are registered with a `Namespaced` webhook `scope`, which means that registered webhooks will forward all requests matching the given `resources` and `operations` to the `AdmissionPolicy` namespace. Otherwise there is no difference between `AdmissionPolicy` and `ClusterAdmissionPolicy`.

Reviewer: I think we might need to reframe this entirely as below 
```suggestion
`AdmissionPolicy` is a namespace-wide resource. This means that its resources are registered with a `namespace` webhook `scope` that will forward all requests matching the given `resources` and `operations` to the `AdmissionPolicy` namespace. Other than that, there are no functional differences between the `AdmissionPolicy` and `ClusterAdmissionPolicy` resources.
```

Author: I would leave the scope as 'Namespaced' since a scope can be 'Cluster' or 'Namespaced' [more info](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#configure-admission-webhooks-on-the-fly)
Other suggestion done in https://github.com/kubewarden/docs/commit/3884c372cb0809d18ac9e102d03aef8387e41992
